backends-clickhouse/src-kafka/main/scala/org/apache/gluten/component/CHKafkaComponent.scala

```diff
@@ -29,6 +28,5 @@ class CHKafkaComponent extends Component {
   override def dependencies(): Seq[Class[_ <: Component]] = classOf[CHBackend] :: Nil
   override def injectRules(injector: Injector): Unit = {
     OffloadKafkaScan.inject(injector)
-    injector.gluten.legacy.injectPost(c => RemoveStreamingTopmostColumnarToRow(c.session, c.caller.isStreaming()))
```

Comment:
Does it nees to remove the rule `RemoveStreamingTopmostColumnarToRow` ?

Reply:
No need, it will used in next pr.